resallocwebui/app.py

```diff
@@ -37,12 +38,20 @@ def pools():
     # e.g. result["copr_hv_x86_64_01_prod"]["STARTING"]
     result = {}
 
+    _, pools_from_config = reload_config()
```

Comment:
please test that this doesn't break selinux (apache is unable to read resalloc's configuration files)

Reply:
It had some issues with `/var/log/resallocserver/main.log` which I fixed with the follow-up commit but surprisingly SELinux has no issues with this (on copr-be-dev)

Reply:
Yeah, I probably mixed up :-) SELinux is likely blocking apache to write into main.log.